.github/workflows/collect-user-submission.py

```diff
@@ -61,8 +60,8 @@ def _create_submission_input(self):
                 authors.append(
                     Author(
                         name=item.get('name', 'anyonymous'),
-                        affiliation=item.get('affiliation'),
-                        affiliation_url=item.get('affiliation_url'),
+                        affiliation=item.get('institution'),
+                        affiliation_url=item.get('institution_url'),
```

Comment:
A thought: should we update the [form template](https://github.com/ProjectPythia/.github/blob/main/.github/ISSUE_TEMPLATE/update-resource-gallery.md) and [card text](https://github.com/ProjectPythia/projectpythia.github.io/blob/b35bfec94c74c926a9a382c2db215e9edb83ce9b/portal/_extensions/gallery_generator.py#L102C49-L102C49) to all use "Affiliation" regardless? It seems more broadly applicable than "Institution" and is in-line with expected fields in eg `CITATION.cff`.

Reply:
Yes, I agree! We can do that as part of this PR

Reply:
Please see [.github/PR#63](https://github.com/ProjectPythia/.github/pull/63)

Reply:
...and also the gallery_generator.py and resource_gallery.yaml files here for "institution" to "affiliation" changes